WORKFLOW_USER_GUIDE.md

```
@@ -843,33 +843,29 @@ The Workflows repository includes maintenance workflows that handle sync, update
-- `python` for Python packages
-- `github-actions` for action updates
-- `security` for security updates
+- `agents:allow-change` (so protected-workflow changes can be reviewed without manual label work)
 
 **Use When:** Automatic, no action needed
 
 ---
 
-### `maint-dependabot-auto-lock.yml` - Lock Dependabot PRs
-**Purpose:** Prevents auto-merge for major version updates
+### `maint-auto-lock-deps.yml` - Auto-lock dependency PRs
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)